facts/Vagrantfile

```diff
@@ -104,6 +104,20 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     host.vm.provision "shell", path: "get_facts.sh"
     host.vm.provision "shell", inline: "/sbin/shutdown -h now"
   end
+  config.vm.define "fedora-22-x86_64" do |host|
+    host.vm.box = "bento/fedora-22"
+    host.vm.provision "shell", inline: "dnf -y install redhat-lsb-core gcc glibc-devel"
```

Comment:
please do not install lsb packages as it generates facts that are not present by defaut on rehat oses